backend/src/services/analysisService.js

```
@@ -0,0 +1,101 @@
+            concentration: {
+                top10: top10,
+                top50: top50,
+                top100: top100,
+            },
+            summary: {
+                totalHolders: holderMetrics.length,
+                currentPrice: currentPrice,
+            }
+        };
+    }
+}
```

Biome (1.9.4)

[error] Avoid classes that contain only static members.

Prefer using simple functions instead of classes with only static members.

(lint/complexity/noStaticOnlyClass)